connexion/exceptions.py

```diff
@@ -1,7 +1,22 @@
+from werkzeug.exceptions import HTTPException, Forbidden, Unauthorized
+
+
 class ConnexionException(Exception):
     pass
 
 
+class ProblemException(ConnexionException, HTTPException):
```

Comment:
Let's expose this in the `connexion.__init__.py` to make it more convenient to import `from connexion import ProblemException` or `import connexion; connexion.ProblemException`.